doc/guidelines/conda/jenkins-conda.rst

```diff
@@ -1,13 +1,12 @@
 .. _Conda-Jenkinsfile:
 
-#################
-Conda Jenkinsfile
-#################
+##################
+Setup Conda for CI
```

Comment:
Would it make sense to call this "Setup Conda for Jenkins"

My understanding (which may be wrong) is that conda/meta.yaml is only for the Jenkinsfile.conda Jenkins job, which I though was not considered as CI-ish as the plain old "Jenkinsfile". Also, it matches the file name.